src/display/annotation_layer.js

```diff
@@ -462,7 +462,7 @@ class TextWidgetAnnotationElement extends WidgetAnnotationElement {
         element.setAttribute("value", textContent);
       }
 
-      element.addEventListener("change", function (event) {
+      element.addEventListener("input", function (event) {
```

Comment:
If this is a problem here, shouldn't it also be changed for checkbox and radio button widgets from previous PRs? If so, please change that in a separate commit.